autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -147,8 +142,8 @@ def predict_proba(self, X: np.ndarray, batch_size: Optional[int] = None) -> np.n
 
         # Neural networks might not be fit to produce a [0-1] output
         # For instance, after small number of epochs.
-        y = np.clip(y, 0, 1)
-        y = sklearn.preprocessing.normalize(y, axis=1, norm='l1')
+        # y = np.clip(y, 0, 1)
+        # y = sklearn.preprocessing.normalize(y, axis=1, norm='l1')
```

Comment:
So the comments above is guaranteed in fact?

Reply:
@ravinkohli